nautobot/docs/installation/nautobot.md

```diff
@@ -17,47 +17,35 @@ These instructions will guide you through the following actions:
 
 ## Choose your `NAUTOBOT_ROOT`
 
-This is where everything related to Nautobot will be installed. We're going to use this value across the documentation. You'll need to set the `NAUTOBOT_ROOT` environment variable to tell Nautobot where to find its files and settings.
+You need to select a directory path where everything related to Nautobot will be installed. We will to use this value across the documentation and will be referred to as `NAUTOBOT_ROOT`. 
```

Comment:
```suggestion
You need to select a directory path where everything related to Nautobot will be installed. We will use this value across the documentation and it will be referred to as `NAUTOBOT_ROOT`. 
```